package.json

```diff
@@ -13,7 +13,8 @@
     "build": "parallel-webpack && npm run bundlewatch",
     "bundlewatch": "bundlewatch --config ./bundlewatch.config.js",
     "doc": "jsdoc -c jsdoc-conf.json",
-    "test": "karma start --single-run --browsers=ChromeHeadless",
+    "pretest": "npm run build && npm run copybuilds",
+    "test": "jasmine test/spec/env/*-spec.js && karma start --single-run --browsers=ChromeHeadless",
```

Comment:
@maoznir is jasmine the only way to do that?
Did we considered git's hooks? https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks

Reply:
That's also an option

Reply:
What so you find better?

Reply:
I think hooks can run this test, but we should still have it as a separate test as here.

Personally I prefer the solution we have in this PR.